ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java

```diff
@@ -522,7 +522,7 @@ public void onResult(@Nullable ThreeDSecureResult threeDSecureResult, @Nullable
                 braintreeClient.sendAnalyticsEvent("three-d-secure.verification-flow.failed");
                 break;
             case CANCEL:
-                callback.onResult(null, new UserCanceledException("User canceled 3DS."));
+                callback.onResult(null, new UserCanceledException("User canceled 3DS.", true));
```

Comment:
This occurs when the Cardinal SDK returns cancellation in the validate response (explicit cancellation).